system_metrics_collector/test/system_metrics_collector/test_linux_cpu_measurement.cpp

```diff
@@ -57,7 +57,7 @@ class TestLinuxCpuMeasurementNode : public system_metrics_collector::LinuxCpuMea
     const std::chrono::milliseconds publish_period)
   : LinuxCpuMeasurementNode(name, measurement_period, publishing_topic, publish_period) {}
 
-  virtual ~TestLinuxCpuMeasurementNode() = default;
+  ~TestLinuxCpuMeasurementNode() override = default;
```

Comment:
This stylistic choice of `clang-tidy`'s (declaring destructors as `override`) is a controversial one: https://github.com/isocpp/CppCoreGuidelines/issues/721.

Reply:
Though I think the majority in the discussion sided with using `override` as being practically good, even if the behavior of a virtual destructor isn't quite the same as that of a virtual function.